### ISSUE TITLE
Summary list component: add ability to add multiple action links to a row

### ISSUE THREAD

**Issue by paulrobertlloyd** [DFE-Digital/govuk-components#218]
The GDS Nunjucks component takes an array of `items` for a row’s `action` links, something this gem currently doesn’t allow.

**Comment by peteryates** [comment]
Just tried doing this with the Nunjucks macros and it looks a bit weird with multiple actions. I amended the code from the first example on the [summary list](https://design-system.service.gov.uk/components/summary-list/) page by adding 'Remove' and 'Confirm':

```javascript
{{ govukSummaryList({
  rows: [
    {
      key: {
        text: "Name"
      },
      value: {
        text: "Sarah Philips"
      },
      actions: {
        items: [
          {
            href: "/change",
            text: "Change"
          },
          {
            href: "/remove",
            text: "Remove"
          },
          {
            href: "/confirm",
            text: "Confirm"
          }
        ]
      }
    },
    // ✂️ 
  ]
}) }}
```

The output definitely looks wrong.

![Screenshot from 2021-07-25 14-08-12](https://user-images.githubusercontent.com/128088/126900189-ce7ac68a-1864-418a-b2fc-e95865db0ef9.png)

I'm either calling it incorrectly or there's an underlying bug. I'll go ahead and implement the component regardless and keep in mind it might need adjusting prior to merge.

**Comment by paulrobertlloyd** [comment]
Interesting. That certainly looks like a bug. I can file an issue on the design system repo, but it does demonstrate that multiple actions is an intentional part of the component, given the separators between each link. Probably not spotted before because it's not often a row has multiple actions associated with it (or the labels are very short when they do).

**Comment by peteryates** [comment]
Fixed by #222
